deploy/cloudformation/organization.yml

```diff
@@ -0,0 +1,88 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Creates IAM roles needed for multi-account access
+Parameters:
+  OrganizationalUnitIds:
+    Type: CommaDelimitedList
+    Description: |
+      Comma-separated list of organizational units to deploy the IAM roles to.
+      Specify the unique IDs of the organizational units where the roles should be deployed.
+      Example: ou-abc123,ou-def456,ou-ghi789
+    AllowedPattern: '.+'
```

Comment:
That's really cool, how is it populated?

Reply:
Essentially the first screenshot. The ids come from the AWS organizations page.